Source/Urho3D/Graphics/ConstantBuffer.h

```diff
@@ -61,6 +61,7 @@ class URHO3D_API ConstantBuffer : public Object, public GPUObject
     /// Return whether has unapplied data.
     bool IsDirty() const { return dirty_; }
 
+    using GPUObject::GetGraphics;
```

Comment:
How do you use it?

Reply:
Introducing getters for core subsystems caused name collision because both `Object` and `GPUObject` now have `GetGraphics()` which returns graphics subsystem. This statement makes compiler prefer `GPUObject::GetGraphics()`. Without this statement compiler produces ambiguity error.